java/code/src/com/suse/manager/webui/services/impl/test/SaltServiceTest.java

```diff
@@ -139,6 +142,45 @@ public void testGenerateSSHKeyExists() throws IOException {
         systemQuery.close();
     }
 
+    @Test
+    public void testLogEntryWithSensitiveData() {
+
+        var saltClient = mock(SaltClient.class);
+        var saltService = new SaltService(saltClient);
+        var listAppender = SaltTestUtils.enableTestLogging(SaltService.class);
+
+        context().checking(new Expectations() {{
+            oneOf(saltClient).call(
+                    with(SaltTestUtils.functionEquals("mgrutil", "check_ssl_cert")),
+                    with(any(Client.class)),
+                    with(any(Optional.class)),
+                    with(any(Map.class)),
+                    with(any(TypeToken.class)),
+                    with(any(AuthMethod.class))
+            );
+            will(returnValue(SaltTestUtils.getCompletionStage(
+                    "/com/suse/manager/webui/services/impl/test/service/ssh_cert_check.json",
+                    new TypeToken<JsonParsingError>() {
+                    }.getType())));
+        }});
+
+        var call = MgrUtilRunner.checkSSLCert(
+                "Certificate:\ncert\n-----BEGIN CERTIFICATE-----\ncert\n-----END CERTIFICATE-----",
+                new SSLCertPair(
+                        "-----BEGIN CERTIFICATE-----\ncert\n-----END CERTIFICATE-----",
+                        "-----BEGIN RSA PRIVATE KEY-----\nkey\n-----END RSA PRIVATE KEY-----"
+                ),
+                null
+        );
+        var result = saltService.callSync(call);
+
+        assertEquals(result, Optional.empty());
+        assertFalse(listAppender.getLog().isEmpty());
+        assertTrue(listAppender.matchInLogs("root_ca=HIDDEN"));
+        assertTrue(listAppender.matchInLogs("server_crt=HIDDEN"));
+        assertTrue(listAppender.matchInLogs("server_key=HIDDEN"));
```

Comment:
Just to be really sure there is not remaining data of the certs, I would assert the data are not present too. May be you'll need to replace them by more recognizable strings too